src/auth/axios.tsx

```diff
@@ -0,0 +1,55 @@
+import axios, { AxiosError, AxiosInstance, AxiosRequestConfig } from 'axios';
+import tokenService from './token';
+import { refresh, logout } from './authAPI';
+
+const Instance: AxiosInstance = axios.create({
+  baseURL: process.env.REACT_APP_API_DOMAIN,
+  timeout: 10000,
```

Comment:
Should this be a constant or part of some properties file? Just a thought, dont think its a big deal either way.

Reply:
Hmmm, yes, going to make it super long on dev to account for debugging active requests.